master/internal/configpolicy/task_config_policy.go

```diff
@@ -24,3 +27,39 @@ type NTSCConfigPolicies struct {
 	InvariantConfig *model.CommandConfig `json:"invariant_config"`
 	Constraints     *model.Constraints   `json:"constraints"`
 }
+
+// PriorityAllowed returns true if the desired priority is within the task config policy limit.
+func PriorityAllowed(wkspID int, workloadType string, priority int, smallerHigher bool) (bool, error) {
+	// Check if a priority limit has been set in task config policies.
+	// Global policies have highest precedence.
+	limit, found, err := GetPriorityLimit(context.TODO(), nil, workloadType)
+	if err != nil {
+		// TODO do we really want to block on this?
```

Comment:
Should we remove this TODO since we discussed with @caehd10 (assuming this Git username is Caetano lol)?